package.json

```diff
@@ -8,7 +8,8 @@
     "deploy": "npm run build && npm test && cd ./lib/flagsmith/ && npm publish && cd ../../lib/react-native-flagsmith && npm publish && cd ../../lib/flagsmith-es && npm publish",
     "deploy:beta": "npm run build && npm test && cd ./lib/flagsmith/ && npm publish --tag beta && cd ../../lib/react-native-flagsmith && npm publish --tag beta && cd ../../lib/flagsmith-es && npm publish --tag beta",
     "build": "rollup -c && node ./move-react.js",
-    "test": "jest --env=jsdom"
+    "test": "jest --env=jsdom",
+    "generatetypes": "npm exec quicktype https://raw.githubusercontent.com/Flagsmith/flagsmith/feat/evaluation-context-schema/sdk/evaluation-context.json -- -o evaluation-context.ts --src-lang schema --just-types --prefer-types"
```

Comment:
TODO: change branch reference to `main` after https://github.com/Flagsmith/flagsmith/pull/4414 is merged.